specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -378,63 +674,360 @@
       },
       "allOf": [
         {
-          "$ref": "#/definitions/AnalyzeTextTask"
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "PiiEntityRecognition"
+    },
+    "AnalyzeTextLanguageDetectionInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/LanguageDetectionAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/LanguageDetectionTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "LanguageDetection"
+    },
+    "AnalyzeTextSentimentAnalysisInput": {
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "parameters": {
+          "$ref": "#/definitions/SentimentAnalysisTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextTask"
+        }
+      ],
+      "x-ms-discriminator-value": "SentimentAnalysis"
+    },
+    "AnalyzeTextJobsInput": {
+      "properties": {
+        "displayName": {
+          "description": "Optional display name for the analysis job.",
+          "type": "string"
+        },
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageAnalysisInput"
+        },
+        "tasks": {
+          "description": "The set of tasks to execute on the input documents.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AnalyzeTextLROTask"
+          }
+        }
+      },
+      "required": [
+        "analysisInput",
+        "tasks"
+      ]
+    },
+    "AnalyzeConversationJobsInput": {
+      "properties": {
+        "displayName": {
+          "description": "Optional display name for the analysis job.",
+          "type": "string"
+        },
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageConversationAnalysisInput"
+        },
+        "tasks": {
+          "description": "The set of tasks to execute on the input conversation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AnalyzeConversationLROTask"
+          }
+        }
+      },
+      "required": [
+        "analysisInput",
+        "tasks"
+      ]
+    },
+    "AnalyzeConversationLROTask": {
```

Comment:
Why distinguish between LRO and non-LRO tasks? Shouldn't they be the same?

Reply:
Even if we perform same operation LRO and non-LRO responses might be different. The LRO task will provide analysis at the conversation level. The non-LRO task will provide a analysis at a conversation item (or turn) level which might be different. Some operations might not be available at a non-LRO level at all and vice versa.